index.js

```diff
@@ -19,13 +19,19 @@ const main = (stream, options) => {
 	}, options);
 
 	let prevLineCount = 0;
+	let prevOut = '';
 
 	const render = (...args) => {
 		if (!options.showCursor) {
 			cliCursor.hide();
 		}
 
 		let out = args.join(' ') + '\n';
+		if (out === prevOut) {
+			return;
+		}
+
+		prevOut = out;
 		out = wrapAnsi(out, getWidth(stream), {
```

Comment:
You have to store the width too, as if the width changes, it should re-render even if the output is unchanged.